site/package.json

```diff
@@ -18,33 +18,34 @@
     "@web3-react/walletlink-connector": "6.2.8",
     "autoprefixer": "10.4.0",
     "big.js": "6.1.1",
-    "dp-auctions-lib": "1.0.0",
-    "erc-20-lib": "1.0.0",
+    "eslint-config-next": "^14.1.3",
     "lodash.debounce": "4.0.8",
     "lodash.orderby": "4.6.0",
     "luxon": "2.1.1",
-    "next": "11.1.2",
+    "next": "^14.1.3",
+    "next-intl": "^3.9.4",
     "next-secure-headers": "2.2.0",
-    "next-translate": "1.2.0",
     "patch-package": "6.4.7",
-    "pf-payment-streams": "1.0.0",
-    "pure-finance-lib": "1.0.0",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
     "react-tooltip": "4.2.21",
-    "swr": "1.0.1",
+    "swr": "^2.2.5",
```

Comment:
It seems due to this update, the API of `useSWR` has [changed](https://swr.vercel.app/blog/swr-v2#fetcher-no-longer-accepts-multiple-arguments) for the `fetcher` function, from accepting multiple arguments to an array of one

From this
```js
useSWR([1, 2, 3], (a, b, c) => {
  assert(a === 1)
  assert(b === 2)
  assert(c === 3)
})
```
to this
```js
useSWR([1, 2, 3], ([a, b, c]) => {
  assert(a === 1)
  assert(b === 2)
  assert(c === 3)
})
```

I believe this will break two custom hooks (for what I could see):
- [useBalance](https://github.com/hemilabs/pure.finance/blob/29bc4dd27c87cbab766b078452e4a41239841117/site/hooks/useBalance.js#L13)
- [useStreams](https://github.com/hemilabs/pure.finance/blob/29bc4dd27c87cbab766b078452e4a41239841117/site/hooks/useStreams.js#L55)

Reply:
good catch! fixed here: https://github.com/hemilabs/pure.finance/pull/1/commits/abcda61f8357cd38f6d8a77b5ddadf6a64b163b9